internal/dag/accessors.go

```diff
@@ -108,15 +108,21 @@ func validateExternalName(svc *core_v1.Service, enableExternalNameSvc bool) erro
 
 // the ServicePort's AppProtocol must be one of the these.
 const (
-	protoK8sH2C = "kubernetes.io/h2c"
-	protoK8sWS  = "kubernetes.io/ws"
+	protoK8sH2C  = "kubernetes.io/h2c"
+	protoK8sWS   = "kubernetes.io/ws"
+	protoHttp    = "www-http"
```

Comment:
Should `www-http` be removed?

It seems to be historical name, as well as `www` which is could also be found in IANA registry, https://datatracker.ietf.org/doc/html/rfc6335#section-5
> By historical accident, the service name "http" has two synonyms
      "www" and "www-http".  When used in SRV records [[RFC2782](https://datatracker.ietf.org/doc/html/rfc2782)] and
      similar service discovery mechanisms, only the service name "http"
      should be used, not these additional names.

Reply:
I didn't see that, thanks !
I'm removing it